site/styles/_footer.scss

```diff
@@ -107,11 +107,14 @@ footer {
 
   a {
     padding: 3px 0 6px 0;
-    margin: 0;
-
-    &:hover .footer-festivalscope-logo {
-      filter: invert(100%);
-    }
+		margin: 0;
+		
+		.footer-festivalscope-logo {
+			filter: invert($footer-fs-invert-percentage);
+		}
+		&:hover .footer-festivalscope-logo  {
+			filter: invert($footer-fs-invert-percentage-hover);
```

Comment:
This can probably be done without new variables.

e.g. $body-color > $body-bg ? invert(100%) : invert(0%)... or something... maybe see what's possible in the sass documentation.

Reply:
I have figured out how to do it without variables. Unfortunatly, you can't use relational operators on colours and the all the methods that could do it are not supported in LibSass yet.

Reply:
This works:
```
&:hover .footer-festivalscope-logo {
      @if lightness($body-color) > lightness($body-bg) {
        filter: invert(0%);
      } @else {
        filter: invert(100%);
      }
    }
```